.ci/magician/exec/exec.go

```diff
@@ -0,0 +1,60 @@
+package exec
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	cp "github.com/otiai10/copy"
+)
+
+type actualRunner struct{}
+
+func (ar *actualRunner) Getwd() (string, error) {
+	return os.Getwd()
+}
+
+func (ar *actualRunner) Copy(src, dest string) error {
+	return cp.Copy(src, dest)
+}
+
+func (ar *actualRunner) RemoveAll(path string) error {
+	return os.RemoveAll(path)
+}
+
+func (ar actualRunner) Chdir(path string) {
+	os.Chdir(path)
+}
+
+func (ar actualRunner) Run(name string, args, env []string) (string, error) {
+	cmd := exec.Command(name, args...)
```

Comment:
can we log the command and output to console?

Reply:
Logging the command could leak our github token.